### PR TITLE
How to collect logs using chectl

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatoliy Bazko <abazko@redhat.com>

### What does this PR do?
Explains how to collect Eclipse Che servers logs using `chectl` tool

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/15485

### CODE REVIEW

**Review by mmorhun — APPROVED** [review]

**Review by slemeur — APPROVED** [review]

File: src/main/pages/che-7/administration-guide/assembly_collecting-logs-using-chectl.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+tags: []
+sidebar: che_7_docs
+permalink: che-7/collecting-logs-using-chectl/
+folder: che-7/administration-guide
+summary:
+---
+
+:parent-context-of-collecting-logs-using-chectl: {context}
+
+[id='collecting-logs-using-chectl']
```
Comment (themr0c) [inline src/main/pages/che-7/administration-guide/assembly_collecting-logs-using-chectl.adoc]:
```suggestion
[id="collecting-logs-using-chectl_{context}"]
```

File: src/main/pages/che-7/administration-guide/assembly_collecting-logs-using-chectl.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+title: Collecting logs using chectl
+keywords:
+tags: []
+sidebar: che_7_docs
+permalink: che-7/collecting-logs-using-chectl/
+folder: che-7/administration-guide
+summary:
+---
+
+:parent-context-of-collecting-logs-using-chectl: {context}
```
Comment (themr0c) [inline src/main/pages/che-7/administration-guide/assembly_collecting-logs-using-chectl.adoc]:
```suggestion
ifdef::context[:parent-context-of-collecting-logs-using-chectl: {context}]
```

File: src/main/pages/che-7/administration-guide/assembly_collecting-logs-using-chectl.adoc [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+[id='collecting-logs-using-chectl']
+= Collecting logs using chectl
+
+:context: collecting-logs-using-chectl
```
Comment (themr0c) [inline src/main/pages/che-7/administration-guide/assembly_collecting-logs-using-chectl.adoc]:
```suggestion
```

File: src/main/pages/che-7/administration-guide/assembly_collecting-logs-using-chectl.adoc [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+It is possible to get all Eclipse Che logs from k8s cluster using `chectl` tool
+
+- `chectl server:start` automatically starts collecting Eclipse Che servers logs during installation of Eclipse Che
+- `chectl server:logs` collects existed Eclipse Che servers logs
+- `chectl workspace:logs` collects running workspace logs
+- `chectl workspace:logs --follow` starts collecting logs of all new workspaces
+
+:context: {parent-context-of-collecting-logs-using-chectl}
```
Comment (themr0c) [inline src/main/pages/che-7/administration-guide/assembly_collecting-logs-using-chectl.adoc]:
```suggestion
ifdef::parent-context-of-collecting-logs-using-chectl[:context: {parent-context-of-collecting-logs-using-chectl}]
ifndef::parent-context-of-collecting-logs-using-chectl[:!context:]
```

**Review by themr0c — COMMENTED** [review]
Thanks for the content.

I added some proposals to get the file with more accurate metadata.

The usage of the `newdoc` tool is encouraged to create new files with accurate matadata:  https://github.com/redhat-documentation/tools/tree/master/newdoc.

**Comment by tolusha** [comment]
@themr0c 
Who is supposed to merge the PR?

**Comment by themr0c** [comment]
I can do it.

**Comment by tolusha** [comment]
@themr0c thank you!

**Pull request merged**